### PR TITLE
Python 2 compatibility

### PR DESCRIPTION
**Pull request by ericenns**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,18 +77,16 @@ cd irida-galaxy-importer
...
-pip3 install bioblend requests-oauthlib
-```
-
-You may need to also install the Python and YAML development libraries. On Ubuntu, you can install them with:
+If you are only running Galaxy on a single machine, please ensure that all the requirements are available with the following commands:
 
 ```bash
-sudo apt-get install python3-dev libyaml-dev
+cd [GALAXY_INSTALL_LOCATION]
+.venv/bin/activate
```
[the remark this answers is not in the document]
Comment (ericenns) [reply README.md:88]: The path is `.venv/bin/activate` or do you mean that I need prepend a `.` with a space

[the remark this answers is not in the document]
Comment (tom114) [reply README.md:88]: Sorry bad comment.  Doesn't the venv need sourced or `.`'d?

So either `. .venv/bin/activate` or `source .venv/bin/activate`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,12 +4,13 @@ All notable changes to irida-galaxy-importer will be documeted in this file.
...
 
-* Upgraded to Python 3
-* Removed toolshed support
+* Added support for Python 3, while maintaining Python 2 support
+* Removed toolshed installation support
 * Dependencies needs are now being addressed with bioconda
 * Changes `--config` option to accept a path to to a config file
 * Adds in `--generate_xml` option which is the new option to generate the galaxy tool xml file
 * Adds in `tool_id`, `tool_file`, and `tool_description` options to the config file. These options are used to customize the galaxy tool xml file (e.g. for multiple installations of the tool for multiple Irida installs)
+* Added in support for importing Assemblies from Irida
```
[the remark this answers is not in the document]
Comment (ericenns) [reply CHANGELOG.md]: Fixed in fae4074

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ericenns) [reply README.md]: Fixed in fae4074

File: UPGRADING.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+<tool file="irida-galaxy-importer/irida_import/irida_import.xml" />
+```
+
+To
+
+```xml
+<tool file="irida-galaxy-importer/irida_import.xml" />
+```
+
+Now we need to regenerate te tool XML file:
```
[the remark this answers is not in the document]
Comment (ericenns) [reply UPGRADING.md]: Fixed in fae4074

File: README.md [hunk 4]
```diff
@@ -77,18 +77,16 @@ cd irida-galaxy-importer
 
 #### 2.1.2. Install dependencies
 
-This tool requires Python 3 and a number of Python libraries. You must make sure these are installed and available on all machines this tool will be run with (e.g., if you are submitting to a cluster, these must be available on all cluster nodes).
+NOTE: This tool supports both Python 2 and Python and 3, though we recommend the use of Python 3 if possible.
 
-If you are only running Galaxy on a single machine, please install **Python 3** and use `pip3` to install the dependencies:
+This tool requires Python and a number of Python libraries. You must make sure these are installed and available within your Galaxies python environment, wherever the tool with be run (e.g., if you are submitting to a cluster, these must be available on all cluster nodes).
```
[the remark this answers is not in the document]
Comment (ericenns) [reply README.md]: Fixed in fae4074

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,12 +4,13 @@ All notable changes to irida-galaxy-importer will be documeted in this file.
 
 ## 2.0.0
 
-* Upgraded to Python 3
-* Removed toolshed support
+* Added support for Python 3, while maintaining Python 2 support
+* Removed toolshed installation support
 * Dependencies needs are now being addressed with bioconda
```
[the remark this answers is not in the document]
Comment (ericenns) [reply CHANGELOG.md]: Fixed in fae4074

**Pull request merged**